studio/schemas/documents/blog.ts

```diff
@@ -54,6 +55,16 @@ const blog = defineType({
         }),
       ],
     }),
+    defineField({
+      name: "redirect",
```

Comment:
maybe make this name less generic
```suggestion
      name: "goToSharedContentLink",
```